### ISSUE TITLE
Hollow Interstage recolor

### ISSUE THREAD

**Issue by AnalogEdition** [KSP-RO/ROTanks#66]
When selecting "Recolor" in the "Core Tex" slider that is available to certain variant selections the default color mask seems to remain below whatever is selected in the Recoloring Gui. The image below illustrates the issue. The upper interstage (paw left of rocket) shows the Recolor Core Tex default color mask. The second interstage (paw right of rocket) has been recolored using the stock black color applied to all sections. The original coloring pattern is discernable. The tank below the second interstage is painted with the same stock black color to further emphasize the difference in shades, it matches neither of the two shades from the interstage. 

![Hollow Interstage recolor issue](https://user-images.githubusercontent.com/13128953/125366777-a66a1100-e32b-11eb-9048-518856072e3e.png)

**Comment by pap1723** [comment]
Yes, this is an issue, it will also be an issue on the newer KatnissTanks as well. It is because of the underlying texture file. The fix for it is to lower the DETAIL % until you get it to match. Short of re-doing all of the underlying texture files (something I am VERY unlikely to ever do) this is the only real fix for this.
